src/map/latent_effect_container.cpp

```diff
@@ -455,14 +455,14 @@ void CLatentEffectContainer::CheckLatentsPartyMembers(size_t members, size_t tru
                     for (size_t m = 0; m < members; ++m)
                     {
                         auto* PMember = (CCharEntity*)m_POwner->PParty->members.at(m);
-                        if (PMember->getZone() == m_POwner->getZone())
+                        if (PMember != nullptr && PMember->getZone() == m_POwner->getZone())
```

Comment:
All of these are the same concept as your other PR: a `static_cast` will just stuff whatever type of pointer it finds into the shape you request, `dynamic_cast` will actually check that it's valid. If a non-player is here (and non-player parties exist, I think) this will force a Mob into the Char pointer.

Reply:
Inline comments: These, as part of the review.

Not inline comments: just comments on the PR

Reply:
My apologies, can you please break it down a bit further for me, and tell me what exactly needs to change here?
Much appreciated.

Reply:
`auto* PMember = dynamic_cast<CCharEntity*>(m_POwner->PParty->members.at(m));